packages/datadog-plugin-promise-js/test/index.spec.js

```diff
@@ -2,4 +2,4 @@
 
 const assertPromise = require('../../dd-trace/test/plugins/promise')
 
-assertPromise('promise-js')
+assertPromise('promise-js', null, '0.0.3 - 0.0.6')
```

Comment:
Version range is not needed since tests should pass even with native.

Reply:
updated, given new tests this isn't required